CMakeLists.txt

```diff
@@ -38,6 +38,18 @@ set(KWIVER_BINARY_DIR           "${CMAKE_BINARY_DIR}")
 set(KWIVER_CONFIG_FILE          "${KWIVER_BINARY_DIR}/kwiver-config.cmake")
 # Create initial setup script
 set(KWIVER_SETUP_SCRIPT_FILE    "${KWIVER_BINARY_DIR}/setup_KWIVER.sh" )
+if(WIN32)
```

Comment:
It looks like we are treating windows as a special case in that we are always generating the shell script, but only generating the batch file if on windows. I think these two cases should be more symmetric. It would be less code to always generate the batch file if the conditionals around the batch file creation are removed and it was always generated. After all, the shell script is generated when built on windows. this would allow us to verify the contents of the batch file when building on Linux and maybe we would remember when we change the shell script to also change the batch file. :-)